py/stream.h

```diff
@@ -43,6 +43,7 @@
 #define MP_STREAM_GET_DATA_OPTS (8)  // Get data/message options
 #define MP_STREAM_SET_DATA_OPTS (9)  // Set data/message options
 #define MP_STREAM_GET_FILENO    (10) // Get fileno of underlying file
+#define MP_STREAM_BUFFER_SIZE   (11) // Get preferred buffer size for file
```

Comment:
I think it should be `MP_STREAM_GET_BUFFER_SIZE`

All the others have either GET or SET in their name, except for `MP_STREAM_TIMEOUT` (but that's because it's both get & set)

I'm not worried about the length of the name.